crates/wdk-build/src/lib.rs

```diff
@@ -286,6 +296,25 @@ impl Config {
         }
     }
 
+    /// Expose `cfg` settings based on this [`Config`] to enable conditional
+    /// compilation. This emits specially formatted prints to Cargo based on
+    /// this [`Config`].
+    fn emit_cfg_settings(&self) {
+        println!(r#"cargo::rustc-check-cfg=cfg(driver_type, values("WDM", "KMDF", "UMDF"))"#);
```

Comment:
This doesn't work in the case where the config fails to be detected. need a way to emit these anyways regardless if metadata is present